lib/helpers/coverage.js

```diff
@@ -35,6 +35,11 @@ module.exports.init = ({
 };
 
 module.exports.setCoverage = ({ path, method, status }) => {
+  if (!schema) {
+    console.warn('Coverage was not initted. If you want to calculate coverage, make sure to call init() on coverage helper. Skipping coverage calculation');
```

Comment:
did you mean `initiated` instead of `initted`?

Reply:
fixed